doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -998,3 +998,25 @@
     // iox::string to std::string
     std::string myConvertedIoxString = iox::into<std::string>(myIoxString);
     ```
+
+44. Move and rename `DeadlineTimer`
+
+    ```cpp
+    // before
+    #include "iceoryx_hoofs/cxx/deadline_timer.hpp"
+    iox::cxx::DeadlineTimer myTimer;
+
+    // after
+    #include "iox/deadline_timer.hpp"
+    iox::deadline_timer myTimer;
+    ```
+
+45. Changed include path of `iox::units::Duration`
```

Comment:
Out of curiosity, is it now `iox::Duration` or still `iox::units::Duration`?

Reply:
The latter, as we hopefully will extend `iox::units` with more SI units.

Reply:
It's similar in the STL with `std::chrono`. For me it's ok to have nested namespace, but I agreed currently it does not make a lot of sense.